### PR TITLE
[O2-2088] Provide a valid AliECS workflow dump from DPL

### PR DESCRIPTION
**Pull request by knopers8**
@ktf @teo 
This is surely not the last fix, but this is already version that is usable (though with some caveats). More fixes will follow after MW3.

### CODE REVIEW

File: Framework/Core/src/O2ControlHelpers.cxx [hunk 1]
```diff
@@ -9,106 +9,327 @@
...
+      ai++;
+    }
+    if (!option) {
+      break;
+    }
+
+    static const std::set<std::string> omitOptions = {
+      "--channel-config", "--o2-control", "--control", "--session", "--monitoring-backend",
+      "-b", "--color", "--infologger-severity", "--infologger-mode", "--driver-client-backend",
+      "--shm-segment-size", "--shm-throw-bad-alloc"};
```
Comment (knopers8) [inline Framework/Core/src/O2ControlHelpers.cxx:183]: @ktf @teo  Can you think of any other options that should be omitted or replaced by AliECS?
Here you can find examples of the full argument lists https://github.com/knopers8/ControlWorkflows/commit/bc5e3590c7d25d9f1eedadbeea8e4641fe1e7ef9

**Comment by ktf** [comment]
not on top of my mind.

**Comment by ktf** [comment]
Nice. Could you please add a test as well, something simple like the DDS one would be great.

**Comment by knopers8** [comment]
Yes, certainly.

**Pull request merged**